api/serializers/job_serializers.py

```diff
@@ -9,9 +9,11 @@ class JobSerializer(serializers.ModelSerializer):
     company_slogan = serializers.CharField(read_only=True, source='company.slogan')
     company_location = serializers.CharField(read_only=True, source='company.location')
     logo_url = serializers.CharField(read_only=True, source='company.logo_url')
+    email = serializers.EmailField(read_only=True, source='company.email')
 
     class Meta:
         model = Job
         fields = ['id', 'created', 'company_id', 'title', 'is_active', 'location', 'description', 'languages', 'skills',
-                  'cultures', 'company_name', 'company_slogan', 'company_location', 'logo_url']
-        read_only_fields = ['id', 'created', 'company_id', 'company_name', 'company_slogan', 'company_location', 'logo_url']
\ No newline at end of file
+                  'cultures', 'company_name', 'company_slogan', 'company_location', 'logo_url', 'email']
+        read_only_fields = ['id', 'created', 'company_id', 'company_name', 'company_slogan', 'company_location',
+                            'logo_url', 'email']
```

Comment:
Please add the trailing blank line.
```suggestion
                            'logo_url', 'email']

```